lib/index.js

```diff
@@ -13,12 +15,20 @@ const linterUiDefault = {
       atom.packages.getLoadedPackage('linter-ui-default').metadata['package-deps'].push('busy-signal')
     }
 
-    if (!atom.inSpecMode()) {
-      // eslint-disable-next-line global-require
-      require('atom-package-deps').install('linter-ui-default', true)
+    let callbackID
+    const installLinterUIDefaultDeps = () => {
```

Comment:
Actually, this doesn't need access to `this`, so it could be done in that manner.